scripts/FlowConvertorDeployment.s.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+// Scripting tool
+import {Script} from "../lib/forge-std/src/Script.sol";
+
+import {FlowConvertor} from "../contracts/FlowConvertor.sol";
+
+contract FlowConvertorDeployment is Script {
+    address private constant TEAM_MULTI_SIG = 0x13eeB8EdfF60BbCcB24Ec7Dd5668aa246525Dc51;
+    // TODO: Fill the address
+    address private constant FLOW = 0x0000000000000000000000000000000000000000;
```

Comment:
Guessing this will be deployed after everything else so we can get this from deployed contracts like we do above..